src/components/bundle.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Tier from './tier.jsx';
+import $ from 'jquery';
+import './index.css';
+
+const awsAddress = 'http://3.15.215.14:3101/';
+
+class Bundle extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bundleId: parseInt(this.props.match.params.bundleId),
+      tier1Id: 1,
+      tier1Cost: 1,
+      tier2Id: 2,
+      tier2Cost: 3, // from Top Contributors
+      tier3Id: 3,
+      tier3Cost: 25,
+    }
+    this.getBundleData = this.getBundleData.bind(this);
+  }
+
+  componentDidMount() {
+    this.getBundleData();
+  }
+
+  getBundleData() {
+    // var url = new URL(window.location);
+    // var bundleId = url.searchParams.get('bundle');
```

Comment:
If these two lines are cruft, they can go.

Reply:
Yup, that's the old way I was doing it. Thanks!